contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -608,6 +610,9 @@ public void run() throws Exception {
                     verifyOrAddJsonStore(url);
                 }
                 if (build) {
+                    log.info("==================================================================================");
```

Comment:
If you must log several lines at once it should be done transactional (i.e. a single `info` call with embedded "\n" newlines), since depending on how the logs are aggregated other threads might log in-between. But I think it's best to leave unnecessary fluff like these separators out, as they make automatic log processing just more difficult.